### PR TITLE
Finish welcome command

### LINKED ISSUE #13: New members
**Describe the feature you'd like**
Notify mods about new members so they can be welcomed

### PR DESCRIPTION
**Pull request by nayabw**
Closes #13 

I have instructed WebStorm to use 4 spaces instead of tabs. We can consult with @hspil and others to decide on an actual style guide later.

### CODE REVIEW

**Comment by nayabw** [comment]
Going to put everything back to tabs and then request a review again tomorrow morning.

File: .gitignore [hunk 1]
```diff
@@ -3,11 +3,13 @@
...
 bot.log
 keep-alive.sh
+/.idea
 
 # Secrests
 /secrets.json
 /secrets.txt
 /secrets_5XB.json
+/secrets_W5AC.json
 /secrests_W5AC.json
```
Comment (nayabw) [inline .gitignore]: Not sure if this is a misspelling in .gitignore of if this is how you have the file @IanDuncanT

Comment (IanDuncanT) [reply .gitignore]: Nope. Just a misspelling.

**Review by IanDuncanT — APPROVED** [review]
Looks good.

**Pull request merged**